_guides/quickstart-app-development.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Quickstart for DHIS2 App Development
+category: guide
+layout: page
+---
+
+# Quickstart for DHIS2 App Development
+
+## Create a new app and connect it to a local DHIS2 instance in 10 minutes or less.
+
+### Introduction
+You can start developing a DHIS2 app on OSX, Windows or Linux using Docker and the DHIS command line interface (CLI). In this guide, you will install the prerequisites, start up a local DHIS2 instance and create a new app. Your app will connect to the local DHIS2 instance.
+
+### Installing prerequisites
+If you are using Debian Linux, 
+1. Follow the [DHIS2 Docker guide](https://developers.dhis2.org/guides/dhis2-docker) to install the prerequisites
+
+If you are using OSX or Windows,
+1. Install [Docker](https://docs.docker.com/get-docker/)
+3. Install [yarn](https://classic.yarnpkg.com/en/docs/install)
+4. From the command line or terminal, install DHIS CLI
+```
+yarn global add @dhis/cli
+```
+Now that you have installed Docker and the DHIS CLI you are ready to start up DHIS2
```

Comment:
Above line 25 I'd add the following line if it's not redundant? 

`**NB**: Check the `d2` command-line tool with `d2 --help` for more information on available commands.`

Reply:
I've kept this out for now in the spirit of simplicity. I think the link to the CLI docs as you suggested @deboragaleano is sufficient. `d2 --help` brings up a lot of commands that a new DHIS app developer may never need know about.

Reply:
You're right! 👌